src/test/java/org/openrewrite/java/migrate/jakarta/FacesManagedBeansRemovedTest.java

```diff
@@ -0,0 +1,103 @@
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate.jakarta;
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.InMemoryExecutionContext;
+import org.openrewrite.config.Environment;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+class FacesManagedBeansRemovedTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec.parser(JavaParser.fromJavaVersion()
+          .classpathFromResources(new InMemoryExecutionContext(), "cdi-api-2.0.SP1", "jsf-api-2.1.29-11", "jakarta.enterprise.cdi-api-4.0.1"))
+          .recipe(Environment.builder().scanRuntimeClasspath("org.openrewrite.java.migrate.jakarta")
+            .build()
+            .activateRecipes("org.openrewrite.java.migrate.jakarta.FacesManagedBeansRemoved"));
+    }
+
+    @Test
+    void updateFacesManagedBean() {
+        rewriteRun(
+          //language=java
+          java("""
+            package com.test;
+                          
+            import javax.enterprise.context.ApplicationScoped;
+            import javax.enterprise.context.RequestScoped;
+            import javax.enterprise.context.SessionScoped;  
+            import javax.faces.bean.ManagedProperty;
+            import javax.faces.bean.NoneScoped;
+            import javax.faces.bean.ViewScoped;
+             
+            @ApplicationScoped
+            @RequestScoped
+            @SessionScoped
+            @ManagedProperty
+            @NoneScoped
+            @ViewScoped
+            public class ApplicationBean2 {
+             
+            }              
+            """, """
+            package com.test;
+              
+            import jakarta.enterprise.context.ApplicationScoped;
+            import jakarta.enterprise.context.Dependent;
+            import jakarta.enterprise.context.RequestScoped;
+            import jakarta.enterprise.context.SessionScoped;
+            import jakarta.faces.annotation.ManagedProperty;
+            import jakarta.faces.view.ViewScoped;
+                          
+            @ApplicationScoped
+            @RequestScoped
+            @SessionScoped
+            @ManagedProperty
+            @Dependent
+            @ViewScoped
+            public class ApplicationBean2 {
+                          
+            }             
+            """
+          )
+        );
+    }
+
+    @Test
+    void noUpdate() {
+        rewriteRun(
+          //language=java
+          java("""
+            package com.test;
+                          
+            import java.lang.annotation.Documented;
+             
+            @Documented
+            public @interface MyCustomAnnotation{
+                       int studentAge() default 18;            
+            }        
+            """
+          )
+        );
+    }
+
```

Comment:
I don't think we need a test here that's completely unrelated, given that we reuse existing Yaml recipes here.
```suggestion
```